bedrock/firefox/templates/firefox/new/privacy-dmt/scene2.html

```diff
@@ -0,0 +1,98 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/scene2.html" %}
+
+{% block experiments %}{% endblock %}
+
+{% block content %}
+<main class="main mzp-t-firefox mzp-t-dark">
+  <section class="status">
+    <div class="mzp-l-content mzp-t-firefox windows-xpvista">
+      <p>{{ _('You’re using an insecure, outdated operating system <a href="%(url)s">no longer supported by Firefox</a>.')|format(url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}</p>
+    </div>
+    <div class="mzp-l-content mzp-t-firefox">
+      {# fallback_url is replaced by the platform download link via JS, but if
```

Comment:
Is it possible to extend the scene2 template here instead of duplicate all the code?

Reply:
Certainly, we'd just need to add some new blocks in scene2.html